examples/creatives/simid_overlay.js

```diff
@@ -39,4 +40,13 @@ class SimidOverlay extends BaseSimidCreative {
     document.getElementById(elementName).addEventListener(
         'click', sendMessageFunction.bind(this));
   }
+
+  sendMessageOnLog_(elementName, message) {
+    const pauseErrorMessage = {
+          message: "WARNING: testing log",
+        };
```

Comment:
Why always pauseError?
I think this would make more sense if you gave an input field, and let people using the ad type in anything to send as a log message

Reply:
I was just testing out the log functionality to see if it works. I can further develop it as you suggest.  Also variable was named pauseError in our sample creative, here it would just be logMessage. Good catch!